"-gui" & "-rfbport" require "wish" installed, or else x11vnc fails to start.
Removing these 2 parameters makes x11vnc works well on poky sato image.


Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Upstream-Status: Inappropriate [configuration]

Index: x11vnc-0.9.12/x11vnc/x11vnc.desktop
===================================================================
--- x11vnc-0.9.12.orig/x11vnc/x11vnc.desktop	2011-03-03 10:33:18.000000000 +0800
+++ x11vnc-0.9.12/x11vnc/x11vnc.desktop	2011-03-03 10:35:20.000000000 +0800
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=X11VNC Server
 Comment=Share this desktop by VNC
-Exec=x11vnc -gui tray=setpass -rfbport PROMPT -bg -o %%HOME/.x11vnc.log.%%VNCDISPLAY
+Exec=x11vnc -bg -rfbport 5900 -o %%HOME/.x11vnc.log.%%VNCDISPLAY
 Icon=computer
 Terminal=false
 Type=Application

